src/value.cpp

```diff
@@ -1154,13 +1154,13 @@ namespace Exiv2 {
         char plusMinus = '+';
         if (time_.tzHour < 0 || time_.tzMinute < 0) plusMinus = '-';
 
-        const int wrote = sprintf(temp,
+        const int wrote = snprintf(temp, sizeof(temp),
                    "%02d%02d%02d%1c%02d%02d",
                    time_.hour, time_.minute, time_.second,
                    plusMinus, abs(time_.tzHour), abs(time_.tzMinute));
 
         assert(wrote == 11);
```

Comment:
I think this should be an `enforce`. An error indicates that the time is completely off (hours > 100 and such).

Reply:
I will replace it.